TimerCalculatorUtility/TimerCalculatorUtility/ViewModels/MenuPageViewModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using TimerCalculatorUtility.Views;
+using Xamarin.Forms;
+using MenuItem = TimerCalculatorUtility.Models.MenuItem;
+
+namespace TimerCalculatorUtility.ViewModels
+{
+    public class MenuPageViewModel
+    {
+        public MenuPageViewModel()
+        {
+            MenuItems = new List<MenuItem>
+            {
+                new MenuItem
+                {
+                    Text = "Timer",
+                    ImageSource = "x_27_512.png",
+                    Command  = new Command(NavigateToTimer)
+                },
+                new MenuItem
+                {
+                    Text = "Calculator",
+                    ImageSource = "Stopwatch_512.png",
+                    Command  = new Command(NavigateToCalculator)
+                },
+            };
+        }
+
+        public List<MenuItem> MenuItems { get; set; }
+        public Command ToTimerCommand { get; set; }
+        public Command ToCalculatorCommand { get; set; }
```

Comment:
Why do you need these ToTimerCommand or ToCalculatorCommand, if you don't use them? Also, best practice is to try to make no setters until there's no need for it.
For your information (FYI), follow YAGNI - do not implement smth. until you really need it.